test/connection.js

```diff
@@ -199,7 +270,11 @@ describe('Hapi server', () => {
         }, (err) => {
 
             expect(err).to.not.exist();
-            expect(server.plugins['hapi-mongodb'].db.options.url).to.equal('mongodb://localhost:27017');
+            const db = server.plugins['hapi-mongodb'].db;
+            expect(db).to.be.instanceof(Mongodb.Db);
+            expect(db.databaseName).to.equal('test');
+            expect(db.topology.s.clonedOptions.host).to.equal('localhost');
```

Comment:
it looks like in https://github.com/mongodb/node-mongodb-native/blob/V2.1.20/lib/db.js#L106
they started filtering the options with only the following keys:
```javascript
var legalOptionNames = ['w', 'wtimeout', 'fsync', 'j', 'readPreference', 'readPreferenceTags', 'native_parser'
  , 'forceServerObjectId', 'pkFactory', 'serializeFunctions', 'raw', 'bufferMaxEntries', 'authSource'
  , 'ignoreUndefined', 'promoteLongs', 'promiseLibrary', 'readConcern', 'retryMiliSeconds', 'numberOfRetries'
  , 'parentDb', 'noListener', 'loggerLevel', 'logger'];
```